In [1]:
import requests




In [2]:
new_data = {
    'userID': 1,
    'id': 1,
    'title': 'Aprendendo Python',
    'body': 'Manipulando informações da api com requests'
}


In [3]:
url ='https://jsonplaceholder.typicode.com/posts'

In [4]:
post_response = requests.post(
    url,
    json=new_data

)

In [5]:
print(post_response.status_code)

201


O status code 201 é uma resposta HTTP que indica que uma requisição foi bem-sucedida e resultou na criação de um novo recurso. Em resumo, quando você recebe o status code 201 em uma resposta HTTP, significa que o servidor recebeu e processou a requisição com sucesso e criou um novo recurso, como uma nova entrada em um banco de dados ou um novo arquivo em um servidor web. Este status é frequentemente usado em operações de criação de recursos em APIs RESTful.

In [6]:
post_response_json = post_response.json()

In [8]:
print(post_response_json)

{'userID': 1, 'id': 101, 'title': 'Aprendendo Python', 'body': 'Manipulando informações da api com requests'}
